lib/get-system-ui-family.js

```diff
@@ -0,0 +1,64 @@
+import * as caniuse from 'caniuse-lite';
+
+const NO = 'No';
+const APPLE_SYSTEM = 'AppleSystem';
+const BLINK_MAC_SYSTEM_FONT = 'BlinkMacSystemFont';
+
+function computePolyfillFlagFromStatus(status) {
+	const flags = [];
+	const statusArr = status.split(' ');
+	for (const status of statusArr) {
+		if (status === 'y') {
+			return [];
+		} else if (status.includes('n')) {
+			flags.push(NO);
+		} else if (status.includes('#1')) {
```

Comment:
This is a neat way to micro-optimize the system font stack, but could you explain what is going on here in a comment? Or, can we just use browser ids and versions?

Reply:
Good comment. I've added in e30ffd5.